styles.css

```diff
@@ -394,6 +394,54 @@ input[type="submit"]:hover {
   cursor: pointer;
 }
 
+@media (max-width: 768px) {
+  .menu-list-items > ul.active {
+    display: block;
+    position: fixed;
+    overflow: auto;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    list-style: none;
+    margin: 0;
+    padding-top: 26px;
+    padding-left: 36px;
+    background-color: #6070ff;
+    mix-blend-mode: multiply;
+  }
```

Comment:
- [ ] [OPTIONAL] Great job with the sidebar design, but some of the elements are overlapping in your design and I'd suggest you fix that for a better UX. ✔
- Here's how it looks:
  ![Inkedfirefox_hDuUzn2tKJ](https://user-images.githubusercontent.com/104968236/232491916-34ece45a-e773-48bc-89ac-a7758e921d28.jpg)
- As you can see from the above the text "Tonic" overlaps with the list items.
- [ ] You can try an easy fix by removing the `mix-blend-mode: multiply;` and adding an `opacity: 0.85` or you can further do more digging and find a way to work with `mix-blend-mode`